applications/display-server/guithread.cpp

```
@@ -12,46 +12,116 @@
 #include <liboxide/threading.h>
 #include <libblight/clock.h>
+#include <setjmp.h>
+#include <sys/mman.h>
+#include <functional>
+
+#ifdef DEBUG
+#define UNW_LOCAL_ONLY
+#include <libunwind.h>
+#include <cxxabi.h>
+void print_trace() {
+    unw_cursor_t cursor;
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_CLASSINFO is a macro then please configure it.

(unknownMacro)